Core/ReferenceResolver/ExpressionLanguageResolver.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Kaliop\eZMigrationBundle\Core\ReferenceResolver;
+
+use eZ\Publish\API\Repository\Values\Content\Content;
+use Kaliop\eZMigrationBundle\API\ReferenceResolverInterface;
+use Symfony\Component\ExpressionLanguage\ExpressionFunction;
+use Symfony\Component\ExpressionLanguage\ExpressionLanguage;
+use eZ\Publish\Core\FieldType;
+
+class ExpressionLanguageResolver extends AbstractResolver
+{
+    protected $referencePrefixes = ['@='];
+
+    /**
+     * @var ReferenceResolverInterface
+     */
+    private $referenceResolver;
+    /**
+     * @var ExpressionLanguage
+     */
+    private $expressionLanguage;
+
+    public function __construct(ReferenceResolverInterface $referenceResolver, ExpressionLanguage $expressionLanguage)
+    {
+        parent::__construct();
+        $this->referenceResolver = $referenceResolver;
+        $this->expressionLanguage = $expressionLanguage;
+    }
+
+    /**
+     * @param string $stringIdentifier
+     * @return mixed
+     */
+    public function getReferenceValue($stringIdentifier)
+    {
+        return $this->expressionLanguage->evaluate(
+            $this->getReferenceIdentifier($stringIdentifier)
+        );
+    }
+
+    private function getExpressionLanguage()
```

Comment:
This method doesn't seem to be called at all, and its contents seem to partly duplicate some code already defined above